src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/SafeExceptionConverter.cs

```diff
@@ -33,7 +35,31 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 
         public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
         {
-            serializer.Serialize(writer, value);
+            var ex = value as Exception;
+            var serializableEx = new Wrapper(ex);
+            serializer.Serialize(writer, serializableEx);
+        }
+
+        ///<remarks>
+        /// This class needs to exist because System.Exception is not marked with as Serializable attribute
+        ///  and old JSON.NET behaviour was incorrectly treating all ISerializable as Serializable
```

Comment:
nit: there is an extra space before `and old`

Reply:
done